localization/english/wc_mottos_l_english.yml

```
@@ -0,0 +1,62 @@
+﻿l_english:
+ # Scripted
+ ### Humans ###
+ ### Azerothian
+ dynn_Wrynn_motto:0 "Patience, Valiance and Faith"
+
+ ###Gilnean
+ dynn_Greymane_motto:0 "For country and for family"
```

Words in vanilla mottos are capitalized. I suggest doing the same.
```suggestion
 dynn_Greymane_motto:0 "For Country and for Family"
```

---

Done in 6ca3560

---

It wasn't just one motto, other mottos also need attention.

---

Redone in dcd775e